sentry-samples/sentry-samples-spring-boot/src/main/java/io/sentry/samples/spring/boot/ProjectController.java

```diff
@@ -0,0 +1,169 @@
+package io.sentry.samples.spring.boot;
+
+import java.nio.file.NoSuchFileException;
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.UUID;
+import java.util.concurrent.atomic.AtomicInteger;
+import org.jetbrains.annotations.NotNull;
+import org.springframework.graphql.data.method.annotation.Argument;
+import org.springframework.graphql.data.method.annotation.BatchMapping;
+import org.springframework.graphql.data.method.annotation.MutationMapping;
+import org.springframework.graphql.data.method.annotation.QueryMapping;
+import org.springframework.graphql.data.method.annotation.SchemaMapping;
+import org.springframework.graphql.data.method.annotation.SubscriptionMapping;
+import org.springframework.stereotype.Controller;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+@Controller
+public class ProjectController {
+
+  //  public ProjectController(final BatchLoaderRegistry batchLoaderRegistry) {
+  //    // using mapped BatchLoader to not have to deal with correct ordering of items
+  //    batchLoaderRegistry.forTypePair(String.class,
+  // Assignee.class).registerMappedBatchLoader((Set<String> keys, BatchLoaderEnvironment env) -> {
+  //      return Mono.fromCallable(() -> {
+  //        final @NotNull Map<String, Assignee> map = new HashMap<>();
+  //        for (String key : keys) {
+  //          if ("Acrash".equalsIgnoreCase(key)) {
+  //            throw new RuntimeException("Causing an error while loading assignee");
+  //          }
+  //          map.put(key, new Assignee(key, "Name" + key));
+  //        }
+  //
+  //        return map;
+  //      });
+  //    });
+  //  }
```

Comment:
Should we remove the comments here?

Reply:
Probably yes, wasn't quite sure if we should keep it as this is similar to what happens under the hood for `@BatchLoader` below but there's some differences and we may have to dig this back up for debugging. So maybe adding another usage of data loader to the schema that uses this  would be better than removing.